_overviews/scala3-migration/incompat-syntactic.md

```diff
@@ -18,8 +18,7 @@ It is worth noting that most of the changes can be automatically handled during
 |--- |--- |--- |--- |
 |[Restricted keywords](#restricted-keywords)||✅||
 |[Procedure syntax](#procedure-syntax)|Deprecation|✅|[✅](https://scalacenter.github.io/scalafix/docs/rules/ProcedureSyntax.html)|
-|[Parentheses around lambda parameter](#parentheses-around-lambda-parameter)||✅|[✅](https://github.com/ohze/scala-rewrites/tree/dotty/#fixscala213parensaroundlambda)|
-|[Open brace indentation for passing an argument](#open-brace-indentation-for-passing-an-argument)||✅||
```

Comment:
Did you remove this line by accident?

Reply:
yes sorry  no doubt that was broken in my struggle to rebase, I'll fix